scripts/UpdateRepos.ps1

```diff
@@ -83,7 +83,14 @@ try {
             & .\run.ps1 upgrade deps --source $Source --id $LineupID --version $LineupVersion --deps-file $depsFile
 
             Invoke-Block { & git @gitConfigArgs add $depsFile $koreBuildLock }
-            Invoke-Block { & git @gitConfigArgs commit --quiet -m "Update dependencies.props`n`n[auto-updated: dependencies]" @GitCommitArgs }
+            try {
+                & git @gitConfigArgs commit --quiet -m "Update dependencies.props`n`n[auto-updated: dependencies]" @GitCommitArgs
```

Comment:
IMO better to do a quick check. The catch my suppress other issues git-commit
```
& git diff --cached --quiet ./
if ($lastExitCode -ne 0) {
   invoke-block { & git commit }
}
else {
  write-host "no changes"
}
```

Reply:
Yeah that's much cleaner.